test/pylib/manager_client.py

```diff
@@ -47,15 +47,26 @@ def __init__(self, sock_path: str, port: int, use_ssl: bool, auth_provider: Any|
         self.ccluster: Optional[CassandraCluster] = None
         self.cql: Optional[CassandraSession] = None
         # A client for communicating with ScyllaClusterManager (server)
-        self.client = UnixRESTClient(sock_path)
+        self.sock_path = sock_path
+        self.clients_pool = {asyncio.get_running_loop(): UnixRESTClient(sock_path)}
```

Comment:
Nitpick: if I understand this patch correctly, you can initialize clients_pool to an empty dictionary and avoid this code duplication - the client() function will create the client the first time it's used.

By the way, also if I understand correctly, it's not really "pool" in the usual sense of the word. So maybe it's better to just call it "clients" or something longer like "client_for_asyncio_loop".

Reply:
here agree. technically it is not a pool.

Reply:
fixed